test/Status.test.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+jest.mock('opn');
+const open = require('opn');
+const status = require('../lib/utils/status');
+
+open.mockReturnValue('foo');
+const catchMock = jest.fn(() => 'foo');
+open.mockReturnValue({
+  catch: catchMock,
+});
+
+const uri = 'http://127.0.0.1';
+const log = {
+  info: jest.fn(),
+  warn: jest.fn(),
+};
```

Comment:
Can we use `just.spyOn` to better reading tests, thanks!

Reply:
We need to create `log` anyway.  Would you prefer:

```javascript
const log = {
  info: () => {}
  warn: () => {}
}

const infoSpy = jest.spyOn(log, 'info')
const warnSpy = jest.spyOn(log, 'warn')
```

Reply:
???